### ISSUE TITLE
Can we use SCA for oligomeric protein ?

### ISSUE THREAD

**Issue by raghav1986** [reynoldsk/pySCA#7]
Main idea of SCA is to use MSA as a foundation stone. My question does it equally reliable for oligomeric protein?

**Comment by chemgeeklian** [comment]
I don't think so if you can't get MSA of oligomeric proteins. BTW SCA is quite sensitive to quality of the data.
